code42.json

```diff
@@ -629,6 +629,242 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+                {
+            "action": "create user",
+            "identifier": "create_user",
+            "description": "Create a new Code42 user account",
+            "type": "contain",
+            "read_only": false,
+            "parameters": {
+                "org_uid": {
+                    "description":  "Org UID to create user in",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "org_id"
```

Comment:
I am guessing that needs to be `org id` instead

Reply:
I don't think anything produces an org id contains type so I'll just remove it for now..